app/controllers/authenticated_controller.rb

```diff
@@ -19,12 +20,6 @@ class AuthenticatedController < ApplicationController
     raise ActiveRecord::RecordNotFound.new
   end
 
```

Comment:
I think making it a `helper_method` from the controller is  more conventional with PaperTrail usage.